### PR TITLE
add becker theme

### PR DESCRIPTION
**Pull request by RobertoASF**
### Prerequisites

- [x] I have read and understood the [contributing guide][CONTRIBUTING.md]
- [x] The commit message follows the [conventional commits][cc] guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)

### Description

[CONTRIBUTING.md]: https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/CONTRIBUTING.md
[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: themes/becker.omp.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          },
+          "style": "plain",
+          "type": "exit"
+        }
+      ],
+      "type": "prompt"
+    }
+  ],
+  "final_space": true,
+  "version": 1
```
Comment (JanDeDobbeleer) [inline themes/becker.omp.json]: @RobertoASF the config is at version 2 right now, can you update oh-my-posh so you can also auto-migrate this theme?

**Comment by RobertoASF** [comment]
se ha añadido un nuevo themepara oh-my-posh

File: themes/becker.omp.json (hunk 1 above)
Comment (RobertoASF) [reply themes/becker.omp.json]: ok i do it now

Comment (JanDeDobbeleer) [reply themes/becker.omp.json]: It seems to not have been migrated so I'll have a look for you on this one.

**Comment by JanDeDobbeleer** [comment]
@RobertoASF this theme is very similar to others we already have and I'm trying to reduce the amount of identical themes in the stack. I might need to create a separate repo for this specifically, as it's getting a bit out of hand.

**Pull request closed without merging**